docs-website/generateDocsDir.ts

```diff
@@ -15,7 +15,7 @@ const GITHUB_EDIT_URL =
 const GITHUB_BROWSE_URL =
   "https://github.com/datahub-project/datahub/blob/master";
 
-const OUTPUT_DIRECTORY = "genDocs";
+const OUTPUT_DIRECTORY = "docs";
```

Comment:
Why do we need to change the location from genDocs to docs?

Reply:
It now generates into docs and then copies into genDocs from the other scripts. This way we don’t have to purge the whole genDocs directory before hot rebuilding.